bemb/model/bemb.py

```diff
@@ -5,13 +5,13 @@
 Update: Apr. 28, 2022
 """
 from pprint import pprint
-from typing import Dict, List, Optional, Union, Tuple
+from typing import Dict, List, Optional, Tuple, Union
 
 import numpy as np
 import torch
 import torch.nn as nn
 from torch_choice.data import ChoiceDataset
-from torch_scatter import scatter_max, scatter_logsumexp
+from torch_scatter import scatter_logsumexp, scatter_max
```

Comment:
have you added automatic linting/styling? this seems to be a change for alphabetical order, for eg.

do I/ a new user has to make sure of this before committing new code?

Reply:
I was using the automatic "sort import statements" function in the VSCode, it seems to be the alphabetical order after the commit since "L" goes before "M".